### PR TITLE
SimakovaKY_lab_2 

### PR DESCRIPTION
**Pull request by KristinaSim**

### CODE REVIEW

File: SimakovaKY/SimakovaKY_lab_2 [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    int max, min;
+    int per = 0;
+    int i, j;
+   
+      max = -1000000;
+      min = 1000000;
+      sr_znach = 0;
+    for (int i = 0; i < size; i++) {
+
+        arr[i] = rand();
```
Comment (Toxa-man) [inline SimakovaKY/SimakovaKY_lab_2:14]: On some platforms rand() can return values more than 1000000 and less than -1000000
Please find another way to set bounds for min and max values

File: SimakovaKY/SimakovaKY_lab_2 [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+       cout << arr[i]<< ' ';  
+    }
+
+ cout << endl;
+    
+      for (j = 0; j < size; j++) {
```
Comment (Toxa-man) [inline SimakovaKY/SimakovaKY_lab_2]: You should use while loop in sorting as described in task

File: SimakovaKY/SimakovaKY_lab_2 [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+    for (i = 0; i < size; i++) {
+        cout << arr[i]<<' ';
+    }
+
+ cout << endl;
+
+      for (i = 0; i < size; i++) {
+            if (arr[i] % 2 != 0) {
+            cout << arr[i] << ' ';
```
Comment (Toxa-man) [inline SimakovaKY/SimakovaKY_lab_2]: You should create new array from arr, not just print odd elements

File: SimakovaKY/SimakovaKY_lab_2 [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+            }
+       }
+
+ cout << endl;
+
+      for (i = 0; i < 7; i++) {
```
Comment (Toxa-man) [inline SimakovaKY/SimakovaKY_lab_2]: You have size constant that contains size of array

**Review by Toxa-man — COMMENTED** [review]
1. Wrong source branch name
2. You should create folder for your lab 2
3. Code issues

File: SimakovaKY/SimakovaKY_lab_2 (hunk 4 above)
Comment (KristinaSim) [reply SimakovaKY/SimakovaKY_lab_2]: Can you explain how to create a folder? I don't understand what i'm doing wrong.

**Pull request closed without merging**